docs/auto_TOC.py

```diff
@@ -28,11 +28,11 @@ def __init__(self, file, name, category, demo):
 category_list = []
 models = []
 #Retrieve the temporary folder containing Data files and batch examples from the text file
-with io.open('../tmp/tmpDocDir.txt','rb') as fpwdTemp:
+with io.open('../src/tmp/tmpDocDir.txt','rb') as fpwdTemp:
 	pwd = (fpwdTemp.readline()).rstrip('\n')
 fpwdTemp.close()
 
-for root, dirs, files in os.walk('../Models'):
+for root, dirs, files in os.walk('../src/Models'):
```

Comment:
This needs to use an OS-independent filesep instead of the backslashes.

Reply:
Agreed. Is Gabriel still involved since he wrote this function?
readthedoc is based on unix system so that would only be a problem if someone want to compile the doc on a windows machine.

Reply:
I don't think Gabriel is still involved no. 

I guess that makes sense – I wasn't aware that the doc got compiled on the readthedoc server, which is on a unix system. So this isn't a big issue now that you mention that!

Reply:
Hi, true, I'm not involved anymore, but I try to keep me updated to where the project is going.
If you still want to get a OS independant separator, you can replace the hardcoded path with a os.path.join function that will generate a path with the OS specific separator like this:
for root, dirs, files in os.walk(os.path.join('..','src','Models')):

Reply:
thanks @gab-berest if you have time please commit these changes! Otherwise no big deal, and all the best for u

Reply:
I don't have really the time. It was only to help you guys.